### PR TITLE
Remove Alluxio EMR scripts

### PR DESCRIPTION
**Pull request by LuQQiu**
Remove the Alluxio EMR scripts, so we don't have to republish every time to update the scripts

### CODE REVIEW

**Comment by LuQQiu** [comment]
@gpang PTAL thanks, this PR needs to go in both master and branch-2.0

**Comment by apc999** [comment]
where will the scripts be served? Can you elaborate in the PR description?

**Comment by LuQQiu** [comment]
@Xenorith Where will the scripts be served?

**Comment by LuQQiu** [comment]
Failed tests: 
alluxio.job.migrate.MigrateIntegrationTest.migrateDirectory[0](alluxio.job.migrate.MigrateIntegrationTest)
  Run 1: MigrateIntegrationTest.migrateDirectory:92
  Run 2: MigrateIntegrationTest.migrateDirectory:92

Tests in error: 
alluxio.server.ft.ZookeeperFailureIntegrationTest.zkFailure(alluxio.server.ft.ZookeeperFailureIntegrationTest)
  Run 1: ZookeeperFailureIntegrationTest.zkFailure:78 » Timeout Timed out waiting for a...
  Run 2: ZookeeperFailureIntegrationTest.zkFailure:78 » Timeout Timed out waiting for a...

**Comment by LuQQiu** [comment]
jenkins, test this please

**Comment by LuQQiu** [comment]
@apc999 PTAL, thanks! Updated the script location

File: docs/en/compute/AWS-EMR.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ To begin with, [download an Alluxio release](https://www.alluxio.io/download) an
 ```console
 $ aws emr create-default-roles
 ```
-2. Download `alluxio-emr.sh` from [our github](https://github.com/Alluxio/alluxio/tree/master/integration/emr/) and
+2. Download `alluxio-emr.sh` from [alluxio public S3 bucket](https://alluxio-public.s3.amazonaws.com/emr/2.0.1/alluxio-emr.sh) and
```
Comment (ZacBlanco) [inline docs/en/compute/AWS-EMR.md]: from **the** alluxio public S3 bucket

**Comment by LuQQiu** [comment]
Failed tests: 
alluxio.job.migrate.MigrateIntegrationTest.migrateDirectory[0](alluxio.job.migrate.MigrateIntegrationTest)
  Run 1: MigrateIntegrationTest.migrateDirectory:92
  Run 2: MigrateIntegrationTest.migrateDirectory:92

Tests in error: 
alluxio.server.ft.ZookeeperFailureIntegrationTest.zkFailure(alluxio.server.ft.ZookeeperFailureIntegrationTest)
  Run 1: ZookeeperFailureIntegrationTest.zkFailure:78 » Timeout Timed out waiting for a...
  Run 2: ZookeeperFailureIntegrationTest.zkFailure:78 » Timeout Timed out waiting for a...

File: docs/en/compute/AWS-EMR.md (hunk 1 above)
Comment (LuQQiu) [reply docs/en/compute/AWS-EMR.md]: Done, thanks Zac

File: docs/en/compute/AWS-EMR.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ To begin with, [download an Alluxio release](https://www.alluxio.io/download) an
 ```console
 $ aws emr create-default-roles
 ```
-2. Download `alluxio-emr.sh` from [our github](https://github.com/Alluxio/alluxio/tree/master/integration/emr/) and
+2. Download `alluxio-emr.sh` from the [alluxio public S3 bucket](https://alluxio-public.s3.amazonaws.com/emr/2.0.1/alluxio-emr.sh) and
```
Comment (apc999) [inline docs/en/compute/AWS-EMR.md]: what about `integration/emr/alluxio-emr.json`, where is this file living now? @LuQQiu

**Comment by LuQQiu** [comment]
@ZacBlanco I change the 
```
aws emr create-cluster --configurations https://alluxio-public.s3.amazonaws.com/emr/2.0.1/alluxio-emr.json
```
will it work?

File: docs/en/compute/AWS-EMR.md (hunk 2 above)
Comment (ns1123) [reply docs/en/compute/AWS-EMR.md]: User's don't need to download from the public bucket unless they want to actually make changes to the bootstrap. They can provide our S3 URI directly and run 'aws emr create-cluster' off of that.

File: docs/en/compute/AWS-EMR.md [hunk 3]
```diff
@@ -71,7 +71,7 @@ $ aws emr create-cluster \
 --bootstrap-actions \
 Path=s3://bucket/path/to/alluxio-emr.sh,\
 Args=[<download-url>,<root-ufs-uri>,<optional-path-to-site-properties>,<optional-additional-delimited-properties>,<optional-property-delimiter>] \
---configurations file://${ALLUXIO_HOME}/integration/emr/alluxio-emr.json \
+--configurations https://alluxio-public.s3.amazonaws.com/emr/2.0.1/alluxio-emr.json \
```
Comment (ZacBlanco) [inline docs/en/compute/AWS-EMR.md]: @ns1123 Wasn't the requirement that it was an s3 URI?

Comment (ns1123) [reply docs/en/compute/AWS-EMR.md]: the json file is a local file. The bootstrap needs to be an S3 URI. So this section does need to be updated as well

Comment (ZacBlanco) [reply docs/en/compute/AWS-EMR.md]: According to `aws emr create-cluster help` that states

```
          Specifies a JSON file that contains  configuration  classifications,
          which you can use to customize applications that Amazon EMR installs
          when cluster instances launch. Applies only to Amazon  EMR  4.0  and
          later.  The  file referenced can either be stored locally (for exam-
          ple, --configurations file://configurations.json ) or stored in Ama-
          zon    S3    (for   example,   --configurations   https://s3.amazon-
          aws.com/myBucket/configurations.json ). Each classification  usually
          corresponds  to  the xml configuration file for an application, such
          as yarn-site for YARN. For a list of available configuration classi-
          fications  and  example  JSON, see the following topic in the Amazon
          EMR Release Guide:
```

The http URL is fine here

Comment (LuQQiu) [reply docs/en/compute/AWS-EMR.md]: Thanks for confirming!

**Comment by apc999** [comment]
LGTM

**Comment by apc999** [comment]
alluxio-bot, merge this please.

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please.

**Pull request merged**